x-pack/legacy/plugins/alerting/server/routes/mute.test.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { createMockServer } from './_mock_server';
+import { muteAlertInstanceRoute } from './mute';
+
+const { server, alertsClient } = createMockServer();
+muteAlertInstanceRoute(server);
+
+test('mutes an alert instance', async () => {
+  const request = {
+    method: 'POST',
+    url: '/api/alert/1/alert_instance/2/_mute',
+  };
+
+  const { statusCode } = await server.inject(request);
+  expect(statusCode).toBe(204);
+  expect(alertsClient.mute).toHaveBeenCalledWith({ alertId: '1', alertInstanceId: '2' });
+});
```

Comment:
Test for nonexistent instance?

Reply:
Since they're de-coupled from alert instance objects. There isn't any checks to see if the alert instance exists or not. This will cause the API to always return 204.